core/domain/stats_domain.py

```diff
@@ -145,6 +145,18 @@ def create_default(cls, exp_id, exp_version, state_stats_mapping):
         """
         return cls(exp_id, exp_version, 0, 0, 0, 0, 0, 0, state_stats_mapping)
 
+    def compute_answer_count(self):
```

Comment:
get_sum_of_first_hit_counts()

(and you can transfer the explanation of "first entries ~ num users" to the caller, since it's only the caller that needs to understand the equivalence, not this method here.)

Also, tests?